spec/features/users/first_time_user_spec.rb

```diff
@@ -13,7 +13,7 @@
       visit '/'
     end
 
-    it 'saves the preferences and shows the landing page', intermittent: 1 do
+    it 'saves the preferences and shows the landing page', intermittent: 2 do
       expect(User.count).to eq(1)
```

Comment:
Was it this line that failed? Are we not correctly clearing the User table somewhere?